### DISCUSSION TITLE
how to hide old news

### DISCUSSION THREAD

**Discussion by mengsong-econ** [alshedivat/al-folio#2488] [Q&A] [answered]
hi everyone, 

I am trying to hide the old news from my home display page of my website. Instead of deleting the old one, is there a way we can hide them

**Comment by george-gca** [comment]
What do you mean by old news?

**Reply by mengsong-econ** [reply]
the news that I do not want to display on the main page but I do not want to delete it

**Reply by george-gca** [reply, marked as answer]
You want to hide the news from the about page right? Just set `news: false` in the [_pages/about.md](https://github.com/alshedivat/al-folio/blob/master/_pages/about.md) page front matter.

**Reply by mengsong-econ** [reply]
Thank you. I want to hide some news from about page not all news. Is there a way we can do ?

**Reply by george-gca** [reply]
No, you have to implement this, but it is not hard to do it. The news in the about page was thought to display the last X news, so we only implemented the limit of news to display there.

To do this, you'll need to add a field in the news front matter to indicate that it will be displayed there. Then, add a verification for this field after this for:

https://github.com/alshedivat/al-folio/blob/1a7fddecf85e03ee6a2663d655cc0f6ccfb5bd34/_includes/news.liquid#L17

**Reply by mengsong-econ** [reply]
thank you! let me try.
